builtin/game/item.lua

```diff
@@ -597,13 +597,11 @@ core.nodedef_default = {
 	-- Node properties
 	drawtype = "normal",
 	visual_scale = 1.0,
-	-- Don't define these because otherwise the old tile_images and
-	-- special_materials wouldn't be read
-	--tiles ={""},
-	--special_tiles = {
-	--	{name="", backface_culling=true},
-	--	{name="", backface_culling=true},
-	--},
+	tiles ={""},
+	special_tiles = {
+		{name="", backface_culling=true},
+		{name="", backface_culling=true},
+	},
```

Comment:
Don't think there's much value in having these stubs (also there can be more than two special tiles, this code is probably old).
Suggestion:
```lua
	tiles = {},
	special_tiles = {},
```